app/src/main/res/values/strings.xml

```diff
@@ -67,29 +67,29 @@
     <string name="recovery_code_input_hint_11">Word 11</string>
     <string name="recovery_code_input_hint_12">Word 12</string>
     <string name="recovery_code_error_empty_word">You forgot to enter this word.</string>
-    <string name="recovery_code_error_invalid_word">Wrong word. Did you mean %1$s or %2$s?</string>
-    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all words and try again!</string>
+    <string name="recovery_code_error_invalid_word">Wrong word. Did you mean \"%1$s\" or \"%2$s\"?</string>
+    <string name="recovery_code_error_checksum_word">Wrong code. Please check all the words and try again.</string>
 
     <!-- Notification -->
-    <string name="notification_channel_title">Backup Notification</string>
+    <string name="notification_channel_title">Backup notification</string>
     <string name="notification_title">Backup running</string>
-    <string name="notification_backup_result_complete">Backup complete</string>
-    <string name="notification_backup_result_rejected">Not backed up</string>
-    <string name="notification_backup_result_error">Backup failed</string>
+    <string name="notification_backup_result_complete">Done</string>
+    <string name="notification_backup_result_rejected">Could not perform backup</string>
+    <string name="notification_backup_result_error">Try again</string>
 
-    <string name="notification_success_title">Backup finished</string>
+    <string name="notification_success_title">Done</string>
     <string name="notification_success_num_not_backed_up">%1$d apps could not get backed up</string>
-    <string name="notification_failed_title">Backup failed</string>
+    <string name="notification_failed_title">That didn\'t go well</string>
 
-    <string name="notification_error_channel_title">Error Notification</string>
-    <string name="notification_error_title">Backup Error</string>
-    <string name="notification_error_text">A device backup failed to run.</string>
+    <string name="notification_error_channel_title">Error notification</string>
+    <string name="notification_error_title">Backup error</string>
+    <string name="notification_error_text">One device was not backed up.</string>
```

Comment:
One doesn't sound right here, since it's not like you can backup multiple devices.

"Backup failed to run." perhaps

Reply:
"Backup failed to run" is very non-descriptive. The result as plainly as possible is what matters.